superset/reports/commands/execute.py

```diff
@@ -147,6 +147,7 @@ def _get_url(
                     "ChartRestApi.get_data",
                     pk=self._report_schedule.chart_id,
                     format="csv",
+                    type="post_processed",
```

Comment:
we should make this an enum later on

Reply:
This is already an enum, actually, let me change the call to use it.